presto-docs/src/main/sphinx/functions/qdigest.rst

```diff
@@ -0,0 +1,49 @@
+=========================
+Quantile Digest Functions
+=========================
+
+These functions allow for creating and querying quantile digests.
+
+Data Structures
+---------------
+
+Presto implements the quantile digest data strucuture as ``qdigest``,
+and it takes a parameter which must be one of ``bigint``, ``double``
+or ``real``.  A ``qdigest`` can be used to store approximate percentile
+information from a data set.  They may also be merged without losing
+precision.
```

Comment:
I would explain here how one could materialize this to storage (by casting to/from varbinary)